core/domain/stats_services.py

```diff
@@ -757,7 +646,8 @@ def delete_playthroughs_multi(playthrough_ids):
     Args:
         playthrough_ids: list(str). List of playthrough IDs to be deleted.
     """
-    stats_models.PlaythroughModel.delete_playthroughs_multi(playthrough_ids)
+    stats_models.PlaythroughModel.delete_multi(
+        stats_models.PlaythroughModel.get_multi(playthrough_ids))
```

Comment:
Should this also be run in a transaction?

Reply:
Done.